snow/networking/sender/sender.go

```diff
@@ -1026,7 +1037,13 @@ func (s *sender) SendPushQuery(ctx context.Context, nodeIDs set.Set[ids.NodeID],
 // the specified chains on the specified nodes.
 // The PullQuery message signifies that this consensus engine would like each
 // node to send their preferred frontier.
```

Comment:
Also, should we just remove the outdated comment, now that chits sends preference, preference at height, accepted?